cloudfoundry-example/README.md

```diff
@@ -0,0 +1,28 @@
+# Vert.x on Cloud Foundry
+
+This demo shows a very simple hello world Vert.x project for [Cloud Foundry](https://www.cloudfoundry.org/). It has a simple HTTP server which simply serves the /webroot/index.html.
+
+This demo uses [Vert.x 3.5.0](http://vertx.io) and is packed using the official [CloudFoundry java-buildpack](https://github.com/cloudfoundry/java-buildpack). The `manifest.yml` specifies the app's memory to be 768MB because any lower and the java-buildpack throws an error that it can't allocate enough heap space.
```

Comment:
Don't use the Vert.x version in the document, we're going to maintain the example with newer versions anyway.

Reply:
Good point